e2e/testcases/gcenode_test.go

```diff
@@ -71,9 +73,23 @@ func TestGCENodeCSR(t *testing.T) {
 		}
 	}`)
 
-	err := nt.WatchForAllSyncs(nomostest.WithRootSha1Func(nomostest.RemoteRootRepoSha1Fn),
+	nt.T.Log("Add the namespace-repo directory to RepoSync's repo")
+	repoSync := fake.RepoSyncObjectV1Beta1(testNs, configsync.RepoSyncName)
+	repoSyncRef := nomostest.RepoSyncNN(testNs, configsync.RepoSyncName)
+	nt.Must(nt.NonRootRepos[repoSyncRef].Copy("../testdata/hydration/namespace-repo", "."))
```

Comment:
I'm curious about the decision to add hydration to these test cases. My understanding was that the primary motivation of this change is to add test coverage for more auth types with RepoSync. Is there also intent to add more test coverage of hydration?

Reply:
Oh I see, this test was already using hydration for RootSync. This seems less unexpected to me in that case

Reply:
Agreed these tests don't require hydration. Historically, we chose an existing CSR repo, which happened to be a DRY repo. Now we can create the CSR repo in a hermetic way, it could be any repo.